core/platform/auth/firebase_auth_services_test.py

```diff
@@ -450,6 +450,30 @@ def assert_is_not_super_admin(self, uid):
         self._test.assertNotEqual(
             custom_claims.get('role', None), feconf.FIREBASE_ROLE_SUPER_ADMIN)
 
+    def assert_is_disabled(self, uid):
+        """Asserts that the given ID is a disabled account.
+
+        NOTE: This method can only be called after the instance has been
```

Comment:
No idea what this means. Could you please elaborate? Instance of what, and what does it mean to "install an instance to a test case"?

Reply:
Done.